src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -23,6 +29,27 @@ Released 2023-May-25
   This also reintroduces the `System.Reflection.Emit.Lightweight` dependency.
   ([#4407](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4407))
 
+* The `OpenTelemetryLoggerOptions.AddOtlpExporter` extension no longer
```

Comment:
This doesn't seem right. We don't want people to expect these features from a `1.5.0-rc.1` version of this package.

I think it's better to simply have one entry in the Unreleased section about merging the Logs package and adding a link to the Logs package's CHANGELOG for reference: https://github.com/open-telemetry/opentelemetry-dotnet/blob/core-1.5.0/src/OpenTelemetry.Exporter.OpenTelemetryProtocol.Logs/CHANGELOG.md